packages/ember-glimmer/tests/integration/helpers/get-test.js

```diff
@@ -0,0 +1,337 @@
+      possibleValues: {
+        colors1: { apple: 'red', banana: 'yellow' },
+        colors2: { apple: 'green', banana: 'purple' }
+      },
+      objectKey: 'colors1',
+      key: 'apple'
+    });
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[green] [green]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'key', 'banana'));
+    this.assertText('[yellow] [yellow]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[purple] [purple]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[yellow] [yellow]');
+  }
+
+  ['@test should be able to get an object value with a GetStream value and GetStream key']() {
+    this.render(`[{{get (get possibleValues objectKey) (get possibleKeys key)}}] [{{if true (get (get possibleValues objectKey) (get possibleKeys key))}}]`, {
+      possibleValues: {
+        colors1: { apple: 'red', banana: 'yellow' },
+        colors2: { apple: 'green', banana: 'purple' }
+      },
+      objectKey: 'colors1',
+      possibleKeys: {
+        key1: 'apple',
+        key2: 'banana'
+      },
+      key: 'key1'
+    });
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[green] [green]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[red] [red]');
+
+    this.runTask(() => set(this.context, 'key', 'key2'));
+    this.assertText('[yellow] [yellow]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors2'));
+    this.assertText('[purple] [purple]');
+
+    this.runTask(() => set(this.context, 'objectKey', 'colors1'));
+    this.assertText('[yellow] [yellow]');
+  }
+
+  ['@test should yield `get`']() {
+    let fooBarInstance;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super();
+        fooBarInstance = this;
+        this.mcintosh = 'red';
+      }
+    });
+
+    this.registerComponent('foo-bar', {
+      ComponentClass: FooBarComponent,
+      template: `{{yield (get colors mcintosh)}}`
+    });
+
+    this.render(`{{#foo-bar colors=colors as |value|}}{{value}}{{/foo-bar}}`, {
+      colors: {
+        red: 'banana'
+      }
+    });
+
+    this.assertText('banana');
+
+    this.runTask(() => this.rerender());
+    this.assertText('banana');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'mcintosh', 'yellow');
+      set(this.context, 'colors', { yellow: 'bus' });
+    });
+    this.assertText('bus');
+
+    this.runTask(() => {
+      set(fooBarInstance, 'mcintosh', 'red');
+      set(this.context, 'colors', { red: 'banana' });
+    });
+    this.assertText('banana');
+  }
+
+  ['@htmlbars should handle object values as nulls']() {
```

Comment:
I *think* https://github.com/emberjs/ember.js/pull/13173/files#r58126138 should fix it?